web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -133,7 +141,7 @@ export const SidebarCardsList = (props: Props): ReactElement => {
           {showEmptyState() && (
             <div className="fdc fac margin-y-3">
               <h3 className="text-normal">{Config.dashboardDefaults.emptyOpportunitiesHeader}</h3>
-              <img src={`/img/signpost.svg`} className="" alt="" />
+              <img src={`/img/for-you-section.svg`} className="" alt="" />
```

Comment:
[sand] I know you didn't write this originally, but this is an opportunity to clean up. We don't need an empty className attribute.
```suggestion
              <img src={`/img/for-you-section.svg`} alt="" />
```

Reply:
From what I understand, the right approach for hiding images from the screen reader is to explicitly set `alt` to be an empty string. So this is good.

[pebble] We'll also want to set `aria-hidden="true"`